<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>711</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1069</width>
    <height>711</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>106900</width>
    <height>71100</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>EasyMail</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>image/dove1.png</normaloff>image/dove1.png</iconset>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>181</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="listWidget">
      <item>
       <property name="text">
        <string>写信</string>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="background">
        <brush brushstyle="NoBrush">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
       <property name="foreground">
        <brush brushstyle="NoBrush">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>image/pencil2.png</normaloff>image/pencil2.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>已发送</string>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>image/sended.png</normaloff>image/sended.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>收信箱</string>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>image/inbox.png</normaloff>image/inbox.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>草稿箱</string>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>image/contacts.png</normaloff>image/contacts.png</iconset>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>821</width>
     <height>671</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="pageCompose">
       <widget class="QLabel" name="labelNewMail">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei Light</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>新邮件</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelTo">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei Light</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>收件方</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditTo">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>40</y>
          <width>721</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="labelSubject">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei Light</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>主题</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditSubject">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>120</y>
          <width>721</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonSend">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>620</y>
          <width>71</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonSave">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>620</y>
          <width>71</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelTo_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei Light</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>抄送</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditTo_2">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>80</y>
          <width>721</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageSended"/>
      <widget class="QWidget" name="pageInbox">
       <widget class="QListWidget" name="listWidgetInbox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>281</width>
          <height>631</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="inboxTextBrowser">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>0</y>
          <width>541</width>
          <height>671</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Reflesh_Button">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>0</y>
          <width>40</width>
          <height>40</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>image/refresh2.png</normaloff>image/refresh2.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="Delete_Button">
        <property name="geometry">
         <rect>
          <x>770</x>
          <y>620</y>
          <width>40</width>
          <height>40</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>image/TRASH BIN.png</normaloff>image/TRASH BIN.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
       <widget class="QListWidget" name="listWidget_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>281</width>
          <height>40</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Inbox</string>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>image/inbox.png</normaloff>image/inbox.png</iconset>
         </property>
        </item>
       </widget>
       <zorder>listWidgetInbox</zorder>
       <zorder>inboxTextBrowser</zorder>
       <zorder>Delete_Button</zorder>
       <zorder>listWidget_4</zorder>
       <zorder>Reflesh_Button</zorder>
      </widget>
      <widget class="QWidget" name="pageDrafts">
       <widget class="QListWidget" name="listWidgetDrafts">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>281</width>
          <height>631</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QTextBrowser" name="draftsTextBrowser">
        <property name="geometry">
         <rect>
          <x>281</x>
          <y>0</y>
          <width>541</width>
          <height>671</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QPushButton" name="Resend_button">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>620</y>
          <width>71</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
       <widget class="QListWidget" name="listWidget_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>281</width>
          <height>40</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Drafts</string>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>image/contacts.png</normaloff>image/contacts.png</iconset>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QWidget" name="pageEmailSent">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>100</y>
          <width>111</width>
          <height>101</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>image/EmailSentIcon.png</pixmap>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>200</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>邮件成功发送</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonBackToInbox">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>300</y>
          <width>141</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>回到收信箱</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonComposeAnotherEmail">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>300</y>
          <width>221</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>编写另一封邮件</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditTo</tabstop>
  <tabstop>lineEditSubject</tabstop>
  <tabstop>pushButtonSend</tabstop>
  <tabstop>pushButtonSave</tabstop>
  <tabstop>listWidget_4</tabstop>
  <tabstop>Reflesh_Button</tabstop>
  <tabstop>listWidgetInbox</tabstop>
  <tabstop>inboxTextBrowser</tabstop>
  <tabstop>Delete_Button</tabstop>
  <tabstop>listWidgetDrafts</tabstop>
  <tabstop>draftsTextBrowser</tabstop>
  <tabstop>Resend_button</tabstop>
  <tabstop>pushButtonBackToInbox</tabstop>
  <tabstop>pushButtonComposeAnotherEmail</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
